defold-spine/editor/src/spineext.clj

```diff
@@ -901,6 +903,12 @@
                                   (validate-model-skin _node-id spine-scene skins skin)))
             (dynamic edit-type (g/fnk [skins] (->skin-choicebox skins))))
   (property create-go-bones g/Bool (default false))
+  (property playback-rate g/Num (default 1.0))
```

Comment:
Is it OK for the playback-rate to be negative? Otherwise we could add property validation here.

```suggestion
  (property playback-rate g/Num (default 1.0)
            (dynamic error (g/fnk [_node-id playback-rate] (validate-model-playback-rate _node-id playback-rate))))
```

We already have other `validate-model-*` functions in the file, so we can just add this next to them:

```clj
(defn- validate-model-playback-rate [node-id playback-rate]
  (validation/prop-error :fatal node-id :playback-rate validation/prop-negative? playback-rate "Playback Rate"))
```
* `validation/prop-negative?` will produce an error if the value is below zero.
* `validation/prop-zero-or-below?` will also produce an error if the value is exactly zero, i.e. "not positive".

And we also need to call it from `produce-model-own-build-errors` so it will be reported as an error when building:
```clj
(g/defnk produce-model-own-build-errors [_node-id default-animation material animations spine-scene skins skin]
  (g/package-errors _node-id
                    (validate-model-material _node-id material)
                    (validate-model-spine-scene _node-id spine-scene)
                    (validate-model-skin _node-id spine-scene skins skin)
                    (validate-model-default-animation _node-id spine-scene animations default-animation)
                    (validate-model-playback-rate _node-id playback-rate)))
```

Reply:
@matgis yes, negative playback is fine